Unstructured-IO/unstructured
# docs: redirect to docs.unstructured.io on github pages

### Summary

Updates GitHub pages to redirect to the new https://docs.unstructured.io page. This will appear on GitHub pages after the next tag.

### Testing

1. From the docs direction, run `make html`. You should not see any errors or warnings
2. Open `unstructured/docs/build/html/index.html`. It should look like the following:
<img width="1512" alt="image" src="https://github.com/Unstructured-IO/unstructured/assets/1635179/077626a5-d88a-467e-9e37-273a92e75d30">
3. Open `unstructured/docs/build/html/404.html`. It should redirect back to `index.html`. Per the [GitHub pages docs](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-custom-404-page-for-your-github-pages-site), that page will get served for 404 errors, meaning any links to old docs pages will redirect to `index.html`, which points users to the new docs page.
